test/run-tests.sh

```diff
@@ -1,11 +1,45 @@
 #!/usr/bin/env bash
-
-# Return failure as soon as a command fails to execute
-
 set -e
 
 cd "$(dirname "$0")"
 
+# Parse arguments and provide usage message #########################
+
+# Provide a usage message
```

Comment:
```suggestion
```

If the exact same comment can be copy+pasted in multiple scripts, it's probably not necessary.